### ISSUE TITLE
Fully kiosk is stuck in loop with webview crashing

### ISSUE THREAD

**Issue by rchiileea** [home-assistant/frontend#21905]
### The problem

after 2024.9 update fully kiosk is causing a webview crash and restarting itself, I reverted back to 2024.8 to test and its fine it is only after the update it happens I removed my backgrounds from dashboard, tried loading overview dash just in case to test possible causes, even on a new blank dash with one button card it will still crash. 

Loads in chrome fine on tablet, I believe the 2024.9 update has broken something in the fully kiosk integration, causing webview to crash

### What version of Home Assistant Core has the issue?

core-2024.9

### What was the last working version of Home Assistant Core?

2024.8

### What type of installation are you running?

Home Assistant OS

### Integration causing the issue

fully kiosk

### Link to integration documentation on our website

https://www.home-assistant.io/integrations/fully_kiosk/

### Diagnostics information

![Screenshot 2024-09-05 083843](https://github.com/user-attachments/assets/c4ca5ee3-16d2-45c5-aee1-c6fa460a2da4)
for some reason getting error on downloading diagnostics

### Anything in the logs that might be useful for us?

```
https://github.com/user-attachments/assets/98fc76de-26c6-4ce7-b44a-893761f3c8e8
```
link to video

### Additional information

issue is fixed by rolliing back to 2024.8

**Comment by rchiileea** [comment]
> Webview crashing wouldn't be related to the Fully Kiosk Browser integration. Gonna move this over to the frontend repo as this is most likely a frontend issue.

I do believe that it is indeed related to that myself, I have gone back to 2024.8 version now as I think it has something to do with the new sections part as if I enable full width and use chrome that sometimes comes back with aw snap error page.

**Comment by rouxdoo** [comment]
I have the same behavior on iPadOS on an iPad Pro that I'm using as a kiosk but not using the Fully Kiosk integration. Sections view crashes Safari and the HA app both.

**Comment by lmsnet** [comment]
I can comfirm this issue, using fullykiosk in nspanel pro and it crashes richt after the update. Before I found this issue on github I even updated the app itself on the device to "Android System WebView_128.0.6613.127_APKPure.apk" and deleted all caches, this didn't help. Reverting back to 2024.8.3 and it worked again.

**Comment by miketague** [comment]
Exactly the same issues on all my Fire tablets Happens in Fully Kios Browser, the HA app and Silk browser. But not Firefox, Firefox seems to load fine. Works fine on both Safari and HA app on my iPhone. Only having the issue when I try to load dashboards using the 'Sections' View

**Comment by silamon** [comment]
I'm going to close this issue as a duplicate of #21900. Please subscribe there to follow updates.
